server/form-pages/apply/risk-and-need-factors/risk-management-features/dateOfOffence.ts

```diff
@@ -82,37 +82,40 @@ export default class DateOfOffence implements TasklistPage {
   renderTableRow(offence: string) {
     return [
       this.textValue(sentenceCase(offence)),
-      this.checkbox(offence, 'current', this.isSelected(offence, 'current')),
-      this.checkbox(offence, 'previous', this.isSelected(offence, 'previous')),
+      DateOfOffence.checkbox(offence, 'current', this.isSelected(offence, 'current')),
+      DateOfOffence.checkbox(offence, 'previous', this.isSelected(offence, 'previous')),
     ]
   }
 
   renderTableBody() {
     return Object.keys(offences).map(offence => this.renderTableRow(offence))
   }
 
+  static checkbox(offence: string, date: 'current' | 'previous', checked: boolean) {
```

Comment:
Would be good if we had a small test for this as we're losing coverage a bit now